doc/pages/Tricks.md

```diff
@@ -78,6 +80,22 @@ $ opam pin <HTTPS clone URL>
 
 Furthermore, `opam update` will then pick up any modification you made to the gist.
 
+##### Strict dependencies
```

Comment:
I am not sure this is the appropriate spot for this, it should rather be in the FAQ

Reply:
It is in the idea of _sharing setup_, that's why I put it here. We can move the entire section in the FAQ.